protocol/wallet.rei

```diff
@@ -1,14 +1,26 @@
 open Mirage_crypto_ec;
-open Helpers;
 
 [@deriving (ord, yojson)]
 type t;
+[@deriving (ord, yojson)]
+type pub_;
+
+let genesis_key: t;
+let genesis_pubkey: pub_;
```

Comment:
Let's go with `secret` for private keys and `key` to public keys. This is how it's used in michelson and it also makes clear, `secret` is a secret.

Reply:
Does this mean we should use `Wallet.secret` instead of `Wallet.t`?

Reply:
A wallet is always a secret, saying `Wallet.secret` would be redundant.